web/apps/labelstudio/src/components/HeidiTips/liveContent.json

```diff
@@ -231,6 +231,18 @@
           "treatment": "enterprise_platform_live"
         }
       }
+    },
+    {
+      "title": "New in Label Studio!",
+      "description": "Now you can use Data Preview to expand, zoom, and pan on images from the grid view. On the Enterprise edition, you can even select and classify multiple images in bulk.",
+      "link": {
+        "label": "Learn more",
+        "url": "https://labelstud.io/learn/categories/release-notes/",
```

Comment:
Waiting on the actual URL

Reply:
https://humansignal.com/blog/bulk-labeling-how-to-classify-in-batches/